src/main/java/randoop/contract/EqualsTransitive.java

```diff
@@ -0,0 +1,62 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract:
+ * <code>!(x0.equals(x1) &amp;&amp; x1.equals(x2))||x0.equals(x2)</code>.
```

Comment:
I had initially written 
`<code>(x0.equals(x1) &amp;&amp; x1.equals(x2)) -> x0.equals(x2)</code>`
but this caused issues with the javadoc since -> wasn't a valid character. Is there a way to write - and > ?